schedule/yast/sle/flows/default_sle15sp3_s390x.yaml

```diff
@@ -1,37 +1,45 @@
 ---
-name: qam-yast_self_update
-description: installation using self_update as boot parameter
-vars:
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp3 with maintenance updates
+bootloader:
   - installation/bootloader_start
+setup_libyui:
   - installation/setup_libyui
+product_selection:
   - installation/product_selection/install_SLES
-  - installation/validate_self_update
+license_agreement:
   - installation/licensing/accept_license
+registration:
   - installation/registration/register_via_scc
+extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
-  - installation/module_registration/add_additional_regcodes
-  - installation/module_registration/import_untrusted_gnpupg_key
+add_on_product:
   - installation/add_on_product/add_maintenance_repos
   - installation/addon_products_sle
+suggested_partitioning:
   - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
   - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_default_target
+software:
+  - installation/select_patterns
+booting:
   - installation/bootloader_settings/disable_boot_menu_timeout
+installation_settings:
   - installation/launch_installation
+installation:
   - installation/confirm_installation
   - installation/performing_installation/perform_installation
+stop_timeout_system_reboot:
+  - installation/performing_installation/stop_timeout_system_reboot_now
+installation_logs:
   - installation/logs_from_installation_system
+confirm_reboot:
   - installation/performing_installation/confirm_reboot
```

Comment:
we need additionally:
` installation/performing_installation/reconnect_after_reboot`
`installation_handel_reboot`
apply the same for SP4.